json-mystique-utils/jackson-utils/src/main/java/com/balajeetm/mystique/util/jackson/convertor/JacksonConvertor.java

```diff
@@ -43,7 +48,13 @@ public class JacksonConvertor implements JsonConvertor {
 
   /** Instantiates a new json jackson convertor. */
   private JacksonConvertor() {
-    objectMapper = new ObjectMapper();
+    this.objectMapper = new ObjectMapper();

```

Comment:
I would suggest the right way would be to use JacksonFactory's object mapper.
Since JacksonFactory's mapper is autowired & registered with mystique modules automatically.

I will reject the PR for now